v0.3/cruorg/components/search/search.css

```diff
@@ -0,0 +1,95 @@
+/* Search styles */
+.cmp-search__field {
+  position: relative; }
+  .cmp-search__field:focus-within .cmp-search__input {
+    outline: none;
+    background-color: #FFFFFF;
+    border-color: transparent;
+    box-shadow: 0 8px 32px rgba(0, 0, 0, 0.08);
+    caret-color: #FFCF07; }
+    .cmp-search__field:focus-within .cmp-search__input::placeholder {
+      color: transparent !important; }
+    .cmp-search__field:focus-within .cmp-search__input::-webkit-input-placeholder {
+      color: transparent !important; }
+    .cmp-search__field:focus-within .cmp-search__input::-moz-placeholder {
+      color: transparent !important;
+      opacity: 1; }
+    .cmp-search__field:focus-within .cmp-search__input:-ms-input-placeholder {
+      color: transparent !important; }
+    .cmp-search__field:focus-within .cmp-search__input:-moz-placeholder {
+      color: transparent !important; }
+  .cmp-search__field:focus-within .cmp-search__icon {
+    background-color: #FFCF07; }
+    .cmp-search__field:focus-within .cmp-search__icon::before {
+      color: #FFFFFF; }
+
+.cmp-search__input {
+  border: #EBECEC 1px solid;
+  border-radius: 24px;
+  font-size: 1rem;
+  line-height: calc(1em + 8px);
+  background-color: transparent;
+  transition: box-shadow .15s ease-in-out, color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out;
+  padding: 11px 64px 11px 24px;
+  width: 100%;
+  box-sizing: border-box; }
+  .cmp-search__input::placeholder {
+    color: #383F43; }
+  .cmp-search__input::-webkit-input-placeholder {
+    color: #383F43; }
+  .cmp-search__input::-moz-placeholder {
+    color: #383F43;
+    opacity: 1; }
+  .cmp-search__input:-ms-input-placeholder {
+    color: #383F43; }
+  .cmp-search__input:-moz-placeholder {
+    color: #383F43; }
+
+.cmp-search__icon {
+  position: absolute;
+  top: 50%;
+  right: 8px;
+  transform: translateY(-50%);
+  font-size: 0;
+  width: 32px;
+  height: 32px;
+  border-radius: 16px;
+  background-color: #EBECEC;
```

Comment:
Do we also need `background-image: none;` to remove the default icon?

Reply:
Good catch. Since the normal markup resource didn't have this component, I was basing my work on [this code](https://github.com/adobe/aem-core-wcm-components/blob/master/content/src/content/jcr_root/apps/core/wcm/components/search/v1/search/search.html) and didn't see how it actually rendered with the default AEM styling.

Reply:
The markup is:
```
<div class="search aem-GridColumn aem-GridColumn--default--12">
<section class="cmp-search" role="search" data-cmp-min-length="3" data-cmp-results-size="10">
    <form class="cmp-search__form" data-cmp-hook-search="form" method="get" action="/content/cru/us/en/metadata/landing.searchresults.json/_jcr_content/content-parsys/search" autocomplete="off" _lpchecked="1">
        <div class="cmp-search__field">
            <i class="cmp-search__icon" data-cmp-hook-search="icon"></i>
            <span class="cmp-search__loading-indicator" data-cmp-hook-search="loadingIndicator"></span>
            <input class="cmp-search__input" data-cmp-hook-search="input" type="text" name="fulltext" placeholder="Search" role="combobox" aria-autocomplete="list" aria-haspopup="true" aria-invalid="false" aria-owns="cmp-search-results-0">
            <button class="cmp-search__clear" data-cmp-hook-search="clear" aria-hidden="true" style="display: none;">
                <i class="cmp-search__clear-icon"></i>
            </button>
        </div>
    </form>
    <div class="cmp-search__results" data-cmp-hook-search="results" role="listbox" aria-multiselectable="false" id="cmp-search-results-0"></div>
    
<script data-cmp-hook-search="itemTemplate" type="x-template">
    <a class="cmp-search__item" data-cmp-hook-search="item">
        <span class="cmp-search__item-title" data-cmp-hook-search="itemTitle"></span>
    </a>
</script>
</section></div>
```

Reply:
At least, locally.

Reply:
Not sure if [this](https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/search/v1/search#bem-description) is also helpful.

Reply:
Yeah, that code you provided matched with the code from the WCM repo. I haven't styled search results because I haven't been given direction on what those should look like.

Reply:
Not sure where the omni-search is going to fit into that either.